### DISCUSSION TITLE
querystring is deprecated now, is there any plan to move away from aws-sdk or to replace aws-sdk

### DISCUSSION THREAD

**Discussion by Ashwini02-Ghaytad** [aws/aws-sdk-js#4309] [Q&A]
While analyzing a issue we observed that `aws-sdk` is pulling `querystring` and `querystring` is deprecated now, is there any plan to move away from `querysting` or to replace `querystring`.

               └─┬ aws-sdk@2.1282.0
                 ├── querystring@0.2.0

**Comment by ajredniwja** [comment]
Hi @Ashwini02-Ghaytad thanks for opening this discussion, there are some modules which have dependency on `queryString` like `url`, just removing would be a breaking change. There is an open PR which removes it but does not handle the dependency part. I have reached out to the team to look into the PR as well. 
The [version 3.x](https://github.com/aws/aws-sdk-js-v3) of the AWS SDK for JavaScript is generally available. For more information see the [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/) or [API Reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/index.html).
